src/main/io/debugserial.c

```diff
@@ -0,0 +1,125 @@
+
+#include "platform.h"
+
+#ifdef USE_DEBUG_SERIAL
+
+#include "io/serial.h"
+#include "common/printf.h"
+#include "io/debugserial.h"
+
+#define INDENTATION_CHAR " "
```

Comment:
Either use char (`' '`) or indentation string of correct length (string of 3 spaces)